index.rst

```diff
@@ -6,65 +6,110 @@
 
-The chiller is controlled via connected to a serial device server (RS-232 to ethernet). The serial device server must run at 9600 baud. 
+The chiller is controlled via a serial device server (RS-232 to ethernet). The serial device server must run at 9600 baud. 
 
-The chiller must be on for a minimum of 15 minutes after the white light source has been turned off before it can be turned off. The white light source cannot be turned on until the chiller is running. 
+The white light source cannot be turned on until the chiller is running. The chiller must be on for a minimum of 15 minutes after the white light source has been turned off before it can be turned off. 
 
 The fluid level of the chiller should be checked occasionally. 
 
 The chiller documentation on Docushare can be found `here <https://docushare.lsst.org/docushare/dsweb/View/Collection-5177>`__
 
+.. note:: The flow of water from the chiller must be kept below 8 GPM. You can read this on the flow meter. To constrict the flow, use the nozzles on the tubes sitting beside the chiller
+
+.. figure:: /_static/flow_meter.png
+   :name: flow_meter
+   :target: ../_images/flow_meter.png
+   :alt: flow_meter
+
+   Flow meter for chiller system. Flow level should be less than 8GPH.
+
 Monochromator
 -------------
-The monochromator is essentially a spectrometer. With the incoming white light, it can output any wavelength from 0 - 1200 nm with fairly high resolution. We will only use it in the range of 350 - 1200 nm. The monochromator is the Horiba iHR 320 Fully Automated Imaging Spectrometer. There are two 1200 gr/mm gratings: one centered at 400 nm (blue) and one at 750 nm (red). These gratings both have a spectral dispersion of :math:`\sim2.31` nm/mm. It also includes a mirror. There are two slits, both of which can be adjusted from 0-5mm, but not including 5mm.  The "Entry" slit is where the light from the white light source enters the monochromator and the "Exit" is where the light exits the monochromator as a "single" wavelength. The monochromator uses very little power at 12V to change the slit width, the wavelength, and the grating. 
+The monochromator is essentially a spectrometer. With the incoming white light, it can output any wavelength from 0 - 1500 nm with fairly high resolution. We will only use it in the range of 350 - 1200 nm. The monochromator is the Horiba iHR 320 Fully Automated Imaging Spectrometer. There are two 1200 gr/mm gratings: one centered at 400 nm (blue) and one at 750 nm (red). These gratings both have a spectral dispersion of :math:`\sim2.31` nm/mm. It also includes a mirror. There are two slits, both of which can be adjusted from 0-5mm, excluding 5mm.  The "Entry" slit is where the light from the white light source enters the monochromator and the "Exit" is where the light exits the monochromator as a "single" wavelength. The monochromator uses very little power at 12V to change the slit width, the wavelength, and the grating. 
```

Comment:
we really only go to ~1050nm. Our chips have zero sensitivity beyond that.

Reply:
changed